pyminio/__init__.py

```diff
@@ -1 +1 @@
-from .main import Pyminio
+from .main import Pyminio, Match, ObjectData, File, Folder
```

Comment:
I hate this name `Match`
tells me nothing if I import it

Reply:
actually it doesnt really make sense for me to import anything from pyminio except for Pyminio itself (and maybe the exception)
y do you allow that?

Reply:
for tests

Reply:
d